testharness.js

```diff
@@ -1583,6 +1585,7 @@ policies and contribution forms [3].
     function Tests()
     {
         this.tests = [];
+        this.promise_tests = Promise.resolve();
```

Comment:
It should be clear to a reader why this exists, and that the intent is that all calls to promise_test will have the tests run sequentially. Either with a comment or renaming the member to e.g.:
promise_test_sequence
sequential_promise_tests
promise_test_chain